apps/app/src/pages/Build/ReviewButton.js

```diff
@@ -24,9 +27,23 @@ export const ReviewButtonBuildFragment = gql`
 export const ReviewButtonRepositoryFragment = gql`
   fragment ReviewButtonRepositoryFragment on Repository {
     permissions
+    private
   }
 `;
 
+export const ReviewButtonOwnerFragment = gql`
+  fragment ReviewButtonOwnerFragment on Owner {
+    consumptionRatio
+  }
+`;
+
+const ReviewChangesButton = React.forwardRef(({ state, ...props }, ref) => (
+  <Button alignSelf="end" ref={ref} {...props}>
+    Review changes
+    <MenuButtonArrow state={state} />
```

Comment:
Without state it is not correct. You have to pass this in children.

Reply:
As I lose too much time with this "use the same button that is not a button" game, I will rollback to two different components. The main purpose was to no duplicate the button label, so I'll use a common variable label.

Reply:
Ok I will do it then.